pkg/fixtures/vulnerability_request.go

```diff
@@ -66,7 +67,7 @@ func GetApprovedDeferralRequest(registry, remote, tag string, cves ...string) *s
 		Id:        uuid.NewV4().String(),
 		Message:   "this is a mock comment by the approver",
 		User:      req.Approvers[0].Clone(),
-		CreatedAt: req.GetCreatedAt().Clone(),
+		CreatedAt: timestamppb.New(req.GetCreatedAt().AsTime()),
```

Comment:
Should we use `protocompat.Clone` here?

Reply:
I have checked how `vtproto` is handling this.
```
(*timestamppb.Timestamp)((*vtTimestamppb.Timestamp)(req.GetCreatedAt()).CloneVT()),
```